backend/vxlan/vxlan.go

```diff
@@ -19,25 +20,28 @@ const (
 	vxlanInterfaceName = "vtep1042"
 	vxlanVni           = 1042
```

Comment:
Can you make this configurable too from catalog and derive vxlanInterfaceName from the VNI?